packages/paste-core/components/combobox/src/styles/ComboboxListbox.tsx

```diff
@@ -1,48 +1,53 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
 
-export interface ComboboxListboxProps {
+export interface ComboboxListboxProps extends Pick<BoxProps, 'element'> {
   children: NonNullable<React.ReactNode>;
 }
 
-const ComboboxListbox = React.forwardRef<HTMLUListElement, ComboboxListboxProps>(({children, ...props}, ref) => {
-  // TODO:
-  // Right now the listbox is absolutely positioned. This will
-  // fail if a combobox is placed at the bottom of a screen
-  // so we need upgrade the positioning using popperjs in the future.
-  return (
-    <Box
-      {...safelySpreadBoxProps(props)}
-      as="div"
-      backgroundColor="colorBackgroundBody"
-      borderRadius="borderRadius20"
-      borderColor="colorBorderWeak"
-      borderWidth="borderWidth10"
-      borderStyle="solid"
-      boxShadow="shadow"
-      listStyleType="none"
-      margin="space0"
-      overflowY="auto"
-      padding="space0"
-      paddingY="space30"
-      position="absolute"
-      width="100%"
-      maxHeight="size30"
-      zIndex="zIndex20"
-      marginTop="space20"
-      ref={ref}
-    >
-      {children}
-    </Box>
-  );
-});
+const ComboboxListbox = React.forwardRef<HTMLUListElement, ComboboxListboxProps>(
+  ({children, element = 'COMBOBOX', ...props}, ref) => {
+    // TODO:
```

Comment:
Not blocking: I see that this TODO comment is not introduced here, but we should consider adding a ticket for this work, or removing these comments.

Reply:
I think this is the ticket for that work: https://issues.corp.twilio.com/browse/DSYS-2794